kibana.yml

```diff
@@ -32,6 +32,8 @@
 # is proxied through the Kibana server.
 # elasticsearch.username: "user"
 # elasticsearch.password: "pass"
+%ELASTICSEARCH_USERNAME%
```

Comment:
(General point being it could be clearer when someone looks at it in a few months time :D)